pam/internal/adapter/nativemodel.go

```diff
@@ -587,7 +587,7 @@ func (m nativeModel) handleFormChallenge(hasWait bool) tea.Cmd {
 	instructions := "Insert '%[1]s' to cancel the request and go back"
 	if hasWait {
 		// Duplicating some contents here, as it will be better for translators once we've them
-		instructions = "Leave the input field empty to wait for other authentication method " +
+		instructions = "Leave the input field empty to wait authentication method " +
```

Comment:
This sentence is missing a word. I suspect you wanted to change it to:

```suggestion
		instructions = "Leave the input field empty to wait for the authentication method " +
```

That makes the next lines redundant though: https://github.com/3v1n0/authd/blob/6a23ceed60cf0cf1117ebf7da861e3ac2e897e38/pam/internal/adapter/nativemodel.go#L592-L595

I'm not sure what the purpose of that if block was and if it's obsolete or not.

Reply:
Yeah, indeed.

Reply:
Also, the wording of "to wait for the authentication method" doesn't seem like a good wording. Maybe "to wait for authentication" would be better, WDYT?

Reply:
Yeah, I agree.

On the duplication, the explanation was at https://github.com/ubuntu/authd/pull/599#discussion_r1810479918